src/twig/partials/footer.twig

```diff
@@ -1,20 +1,32 @@
-<footer class="o-footer color-dark-background screen-tablet:layout--three-columns-gutter">
-  <nav>
-    <a class="o-footer__nav-link" href="overview">Overview</a>
-    <a class="o-footer__nav-link" href="getting-started">Getting Started</a>
-    <a class="o-footer__nav-link" href="feedback-and-support">Feedback and Support</a>
-    <a class="o-footer__nav-link" href="terms-of-service">Terms of Service</a>
-  </nav>
-
-  <div class="o-footer__info">
-    <a href="http://nyc.gov/opportunity" title="{{ package.author }}">
-      <svg aria-hidden="true" class="icon-logo-secondary o-footer__logo text-base-white">
-        <use xlink:href="#icon-logo-secondary"></use>
-      </svg>
-    </a>
-
-    <div id="markdown-about-nyc-opportunity"></div>
-
-    <small>© City of New York, {{ 'now'|date('Y') }} All Rights Reserved.<br>NYC is a trademark and service mark of the City of New York.</small>
+<footer class="o-footer color-dark-background border-navy-70t">
+  <div class="layout-wrap mx-auto">
+    <div class="o-footer__divider border-navy-70t desktop:flex desktop:layout-sidebar mb-4">
+      <div class="pb-4">
+        <a class="block" href="http://nyc.gov/opportunity" title="NYC Opportunity&lt;products@nycopportunity.nyc.gov&gt;">
+          <svg aria-hidden="true" class="o-footer__logo">
+            <use xlink:href="#icon-logo-secondary"></use>
+          </svg>
+          <span class="sr-only">The Mayor's Office for Economic Opportunity</span>
+        </a>
+      </div>
+      <div class="pb-4 desktop:pl-4 desktop:flex justify-end items-center">
+        <nav class="o-footer__inline-nav">
+          <a href="index">Home</a>
+          <a href="https://github.com/CityOfNewYork/nyco-patterns">GitHub Repository</a>
+          <a href="https://github.com/CityOfNewYork/nyco-patterns-framework">NYCO Patterns Framework</a>
+          <a href="https://github.com/topics/nycopportunity?q=org%3Acityofnewyork+topic%3Anycopportunity&amp;unscoped_q=org%3Acityofnewyork+topic%3Anycopportunity">NYCO Opportunity on GitHub</a>
+        </nav>
```

Comment:
@theasteve These links are only relevant to NYCO patterns. We need to swap these footer links with the links from the original footer.

Reply:
Ah didn't catch that, will make changes now https://giphy.com/gifs/afv-funny-fail-lol-3o7ZesJdQccI0sO3fy